packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx

```diff
@@ -66,6 +67,16 @@ interface ContainerProps {
 const Container = ({ title, description, cta, dataTestBase }: ContainerProps) => {
     const { isLocked } = useDevice();
     const actions = Array.isArray(cta) ? cta : [cta];
+    const isOnline = useSelector(state => state.suite.online);
+    const retryButtonRef = useRef<HTMLButtonElement | null>(null);
+
+    useEffect(() => {
+        //TODO: pass exception type and check for specific exception, where we want to do the forced refresh
+        console.log('IS ONLINE: ', isOnline);
+        if (isOnline && retryButtonRef && retryButtonRef.current !== null) {
+            retryButtonRef.current.click();
+        }
+    }, [isOnline]);
```

Comment:
this sounds anti-patternish to me. shouldn't retry be handled somewhere on action/middleware level?